src/App.js

```diff
@@ -1,6 +1,10 @@
+import Summary from './views/Summary';
+
 function App() {
   return (
-    <h1>Hello World</h1>
+    <>
+      <Summary />
+    </>
```

Comment:
Makes totally sense, yes.

Reply:
Matt I would leave this to you because it `addExpenseForm` will require a different container and it is easier if you make the change once you create the `Expense` view

Reply:
Sounds good!